KubeArmor/monitor/logUpdate.go

```diff
@@ -55,6 +56,10 @@ func (mon *SystemMonitor) BuildLogBase(eventID int32, msg ContextCombined, readl
 	log := tp.Log{}
 
 	timestamp, updatedTime := kl.GetDateTimeNow()
+	username, err := user.LookupId(fmt.Sprint(msg.ContextSys.UID))
+	if err != nil {
+		mon.Logger.Errf("error occured when fetching user %e", err)
+	}
```

Comment:
How's this going to work?
1000 could be named postgres in a postgres container but it might not exist on the host KubeArmor is in?